core/src/main/scala/akka/kafka/javadsl/Consumer.scala

```diff
@@ -88,5 +93,49 @@ object Consumer {
       .mapMaterializedValue(new WrappedConsumerControl(_))
       .asJava
 
+  /**
+   * The `plainPartitionedSource` is a way to track automatic partition assignment from kafka.
+   * When topic-partition is assigned to a consumer this source will emit tuple with assigned topic-partition and a corresponding source
+   * When topic-partition is revoked then corresponding source completes
+   */
+  def plainPartitionedSource[K, V](settings: ConsumerSettings[K, V], subscription: AutoSubscription): Source[Pair[TopicPartition, Source[ConsumerRecord[K, V], NotUsed]], Control] = {
+    scaladsl.Consumer.plainPartitionedSource(settings, subscription)
+      .map {
+        case (tp, source) => Pair(tp, source.asJava)
```

Comment:
No, `map { Pair(_, _) }` requires two arguments, but we have just one (`Tuple2`)